go/vt/wrangler/testlib/rpc_reparent_external_test.go

```diff
@@ -292,6 +292,10 @@ func TestRPCTabletExternallyReparentedFailedOldMaster(t *testing.T) {
 		"START SLAVE",
 	}
 
+	// On the oldMaster, we expect a ChangeType to REPLICA.
```

Comment:
Why were these changes needed?

Reply:
Because we're now using `agent.ChangeType` instead of directly updating the topo. So, the ActionAgent needs to be initialized for it to process `ChangeType` properly.

Reply:
You could revert all changes to `rpc_external_reparent.go` and this test and delete the functions in that file + tests either in this PR or in a subsequent one. It doesn't make sense to make changes to it when we should be deleting it.

Reply:
It's fine. I'll ping you separately for the cleanup.